### ISSUE TITLE
[accessibility] Login button does not have a reasonable accessible name

### ISSUE THREAD

**Issue by sandbergja** [metacpan/metacpan-web#3105]
**Describe the bug**

The Login button does not have a usable accessible name (it is the unicode U+F078 character, which is considered "For Private Use").  This means that certain assistive technologies (like screen readers) cannot communicate what the button does.  Other assistive technologies (like voice control) cannot let users efficiently choose the button by name.

**To Reproduce**

1. In Firefox, go to https://metacpan.org/
2. Right click on the Login button in the header
3. Select "Inspect Accessibility Properties"
4. This will open a tree view with the element type and the accessible name.  Note that the accessible name for the button is U+F078.

**Expected behavior** 
The accessible name should be "Login", to clearly and concisely describe what the button does.

**Desktop (please complete the following information):**

- OS: Mac OS
- Browser: Firefox
- Browser version: 127

**Implementation notes**

A simple way to fix this would be to add an `aria-label` attribute to the button with the value `Login`.
